x-pack/plugins/xpack_main/server/lib/telemetry/local/get_kibana.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get, omit } from 'lodash';
+
+export function handleKibanaStats(server, response) {
+  if (!response) {
```

Comment:
I'm kind of curious how we'd get in a state where `response` is validly `null` or `undefined`. I could reasonably see it missing certain usage stats (e.g., from disabled or unused plugins), but not missing core data.

Reply:
The reason for this early return is to avoid throwing exceptions for unit tests that don't pass in mock response for the `handleLocalStats` tests: https://github.com/elastic/kibana/pull/26496/files#diff-46ad76ce05cd14b05bdc60fc08f0001dR104

Reply:
I'm not sure if that's a good thing though. We're allowing for a broken state of Kibana to simplify a few tests, which means we'll have to get luckier to catch it.

Reply:
How about add logging with a warning here?
```
server    log   [17:43:40.419] [warning][local-stats][telemetry] No Kibana stats returned from usage collectors
```

Reply:
5c7036a